src/components/Error/index.js

```diff
@@ -13,7 +13,7 @@ const GordonError = ({ error, errorInfo }) => {
         </Typography>
         <Typography variant="h5" gutterBottom>
           Please email the below error to
-          <a href={"mailto:cts@gordon.edu?subject=Gordon 360 Error"}> cts@gordon.edu</a>.
+          <a href={'mailto:cts@gordon.edu?subject=Gordon 360 Error'}> cts@gordon.edu</a>.
```

Comment:
When a prop is just a string, you can use this simpler syntax:
```suggestion
          <a href="mailto:cts@gordon.edu?subject=Gordon 360 Error"> cts@gordon.edu</a>.
```